book/src/language-tour/check.md

```diff
@@ -1 +1,8 @@
 # Check
+
+Check is faster than assert but has weaker guarantees.
```

Comment:
it's actually the other way around. Check is the less efficient but stronger guarantee one